### PR TITLE
YM-279 YM-417 | Add notification template translations

### PR DESCRIPTION
**Pull request by charn**
This PR adds the notification template translations from open-city-profile production.

### CODE REVIEW

File: youths/utils.py [hunk 1]
```diff
@@ -68,77 +68,170 @@ def delete_contact_persons(youth_profile: YouthProfile, data):
...
+                "förmånerna finns på adressen https://jassari.munstadi.fi/sv/.\r\n\r\nDet här meddelandet skickades "
+                "automatiskt från våra system. Svara inte på det här meddelandet eftersom svaren inte kommer att "
+                "behandlas."
+            ),
+        },
+        "en": {
+            "subject": "{{ youth_profile.approver_first_name }} has approved your Youth Services membership",
+            "html": (
+                "Welcome to the Youth Services of the City of Helsinki! {{ youth_profile.approver_first_name }} has "
+                "approved your membership. Your digital membersip card can be found here: "
```
Comment (tuomas777) [inline youths/utils.py:205]: Typo `membersip`

File: youths/utils.py [hunk 2]
```diff
@@ -68,77 +68,170 @@ def delete_contact_persons(youth_profile: YouthProfile, data):
...
+                '<a href="https://jassari.hel.fi">https://jassari.hel.fi</a><br /><br />For more information on the '
+                "Youth Services membership and its perks, please visit "
+                '<a href="https://jassari.munstadi.fi/en/">https://jassari.munstadi.fi/en/</a><br /><br />'
+                "<i>This message was sent automatically from our system. Please do not reply to this message as the "
+                "replies will not be processed.</i>"
+            ),
+            "text": (
+                "Welcome to the Youth Services of the City of Helsinki! {{ youth_profile.approver_first_name }} has "
+                "approved your membership. Your digital membersip card can be found here: "
```
Comment (tuomas777) [inline youths/utils.py:214]: Ditto

File: youths/utils.py (hunk 1 above)
Comment (charn) [reply youths/utils.py:205]: Nicely spotted. :+1:

**Review by klempine — APPROVED** [review]
👍

**Pull request merged**